src/test/deploy/hosting/convertConfig.spec.ts

```diff
@@ -59,39 +61,152 @@ describe("convertConfig", () => {
     {
       name: "checks for function region if unspecified",
       input: { rewrites: [{ glob: "/foo", function: { functionId: FUNCTION_ID } }] },
-      want: { rewrites: [{ glob: "/foo", function: FUNCTION_ID, functionRegion: "us-central1" }] },
-      existingBackend: backend.of(endpoint({ region: "us-central1" })),
+      want: {
+        rewrites: [
+          {
+            glob: "/foo",
+            function: FUNCTION_ID,
+            functionRegion: "us-central2",
+          },
+        ],
+      },
+      functionsPayload: {
```

Comment:
Ooof. Sorry you had to re-add these test fixtures after my rush/foolishness.

Reply:
Np, happens